assets/i18n/id.json

```
@@ -0,0 +1,262 @@
+    "dgc-quick-desc": "Dogecoin adalah mata uang digital peer-to-peer terdesentralisasi yang memungkinkanmu mengirim uang secara daring dengan mudah. Anggap saja sebagai \"mata uang internet.\"",
+    "dgc-get-started-now": "Mulai Sekarang", 
+    "dgc-learn-more": "Pelajari Lebih Lanjut", 
+    "dgc-what-with-dog": "Ada apa dengan Dogecoin dan anjingnya?", 
+    "dgc-doge-explained": "\"Doge\" sesungguhnya adalah guyonan kami, yang sekaligus menjadi maskot kami juga! Shiba Inu adalah jenis anjing Jepang yang populer sebagai <i>meme</i> dan mewakili Dogecoin.", 
+    "dgc-doge-learn": "Pelajari Lebih Lanjut Tentang Doge", 
+    "dgc-shiba-inu": "Shiba Inu", 
+    "dgc-fun-currency": "Mata uang internet yang asik dan ramah.", 
+    "dgc-set-apart": "Hal yang membedakan Dogecoin dari mata uang digital lainnya adalah komunitas yang luar biasa antusias dan bersemangat, yang terdiri dari orang-orang yang ramah seperti kamu.",
+    "dgc-reddit": "Komunitas Reddit", 
+    "dgc-dogeducation": "Komunitas Dogeducation di Reddit",
+    "dgc-twitter": "Twitter",
+    "dgc-getting-started": "Bagaimana cara memulai menggunakan Dogecoin", 
+    "dgc-setup-wallet": "Membuat sebuah dompet Dogecoin itu <strong>gampang banget</strong>, asli! Periksa panduan memulainya dibawah.", 
+    "dgc-edu-help": "Kalau kamu butuh bantuan, tanyakan di Reddit pada r/dogeducation! <br /> Kita adalah sekelompok orang yang akan membantu dan menjawab setiap pertanyaan yang Kamu ajukan!",
+    "dgc-footer-waiting": "<strong>Jadi tunggu apa lagi?</strong> Mulai pakai Dogecoin hari ini!", 
+    "dgc-footer-getting-started": "Memulai.", 
+    "dgc-footer-learn-how": "Pelajari bagaimana cara menggunakan Dogecoin.", 
+    "dgc-community-title": "Komunitas.", 
+    "dgc-community-participate": "Berpartisipasi dalam Komunitas.", 
+    "dgc-get-dogecoin-title": "Dapatkan Dogecoin.", 
+    "dgc-get-dogecoin-how": "Cari tau bagaimana cara mendapatkan Dogecoin.",
+    "dgc-stores-title": "Toko.", 
+    "dgc-stores-buy": "Beli dengan Dogecoin.",
+    "dgc-footer-credits": "Dibuat Oleh <a href=\"https://ummjackson.com\" target=\"_blank\">Jackson Palmer</a> & <a href=\"https://twitter.com/BillyM2k\" target=\"_blank\">Shibetoshi Nakamoto</a>. Situs Web dibuat oleh <a href=\"https://www.reddit.com/user/edswf1\" target=\"_blank\">/u/edswf1</a>. Logo dibuat oleh <a href=\"http://christinemix.com\" target=\"_blank\">Christine Ricks</a>.",
+    "dgc-footer-copyright": "© The Ðogecoin Project", 
+    "dgc-browser-modal-title": "PILIH SEBUAH DOMPET DIGITAL:", 
+    "dgc-browser-modal-online": "Dompet digital adalah cara tercepat dan termudah untuk menggunakan Dogecoin, tapi kurang aman jika disimpan pada komputer lokal mu. <br /><br />Kami merekomendasikanmu menyimpan Dogecoin dalam jumlah sedikit saja jika menggunakan dompet digital.  <br />Terdapat beberapa dompet berbasis peramban yang tersedia untuk kamu pakai:",
+    "dgc-browser-modal-dogechain": "Domet DogeChain", 
```

Is there any particular reason using Domet instead of Dompet? i can't find them on online KBBI

https://jagokata.com/arti-kata/domet.html

---

I think it's a typo, fix it real quick!

---

```suggestion
    "dgc-browser-modal-dogechain": "Dompet DogeChain", 
```
Like this? Sorry it's originally someone else's work, I'm just managing the process.

---

Thanks, i appreciate it 😄